src/global/app.css

```diff
@@ -77,4 +68,9 @@ html {
 _::-webkit-full-page-media, _:future ion-header .header-background,
 :root .safari_only ion-header .header-background {
   backdrop-filter: none !important;
-}
\ No newline at end of file
+}
+
+:root.ios {
+  --ion-padding: 20px;
+  --ion-margin: 20px;
+}
```

Comment:
This was part of the now deleted `default.css` but does not appear to be related to the light theme